modules/ROOT/pages/dw-core-functions-match.adoc

```diff
@@ -13,7 +13,7 @@ capture groups. Returns the results in an array.
 
 
 Note that you can use `match` for pattern matching expressions that include
-https://docs.mulesoft.com/mule-runtime/4.3/dataweave-pattern-matching[case
```

Comment:
TODO: This was changed to "latest" in the latest source. 

* Note that you can use `match` for pattern matching expressions that include
* https://docs.mulesoft.com/mule-runtime/latest/dataweave-pattern-matching[case
* statements].